update ParaView app to include VUB (#526)
hpcleuven/VscDocumentation: source/compute/portal/ondemand/desktop.rst

       * Tier-2 :ref:`Anansi <Anansi cluster>`
       * Tier-2 :ref:`Hydra <Hydra cluster>`

For improved graphics performance, we recommend the following workflow:

> #. Launch the app in the ``Anansi`` cluster and request a GPU or GPU shard.

Reviewer: the resource selector has been changed to show percentage of GPU, `shard` is not mentioned in ondemand.
```suggestion
#. Launch the app in the ``Anansi`` cluster and request some fraction of GPU.
```
